app/models/forms/affiliation.rb

```diff
@@ -0,0 +1,41 @@
+module Forms
+  class Affiliation < Base
+    # Indicates that this is a deposit, and therefore should be fully validated.
+    attr_accessor :_deposit
+    attr_accessor :id
+    attr_accessor :label
+    attr_accessor :uri
+    attr_accessor :department
+    attr_accessor :abstract_contributor
+    attr_accessor :_destroy
+
+    validates :label, presence: true, if: :_deposit
```

Comment:
Notice that validation is triggered by the _deposit field. This allows for conditional validation (no validation when saving draft; only fully validate when depositing).